packages/orchestration/src/exos/packet-tools.js

```diff
@@ -308,6 +309,17 @@ export const preparePacketTools = (zone, vowTools) => {
         },
       },
       packetWasSentWithMetaWatcher: {
+        /**
+         * @param {{
+         *   eventPattern: Pattern;
+         *   resultV: Vow<unknown>;
+         *   meta: Record<string, any>;
```

Comment:
too cumbersome to be accurate here?

```suggestion
         *   meta: Record<string, unknown>;
```

Reply:
Yeah, when I had it as `unknown` it was pretty cumbersome.